templates/misc/message.part.html

```diff
@@ -121,25 +121,22 @@
                         </li>
                     {% endif %}
                 {% endif %}
-                {% if not is_mp %}
-                    {% if message.is_visible != False or perms_change %}
-                        <li>
-                            <a href="#signal-message-{{ message.id }}" class="ico-after alert open-modal">
+                {% if not is_mp and message.is_visible != False %}
```

Comment:
La condition d'avant c'était :
* ce n'est pas un mp
* **et** le message est visible ou le membre a la permission (notamment le stafff).

La condition que tu as là c'est : 
* ce n'est pas un mp
* **et** le message est visible

Quand tu fais ça, le bouton signaler disparaît pour tout le monde, y compris le staff. Ce qu'on veut c'est juste :
* ce n'est pas un mp

Reply:
Donc faudrait juste que je retire `and message.is_visible != False %` et que je le passe en `True` pour permettre a tout le monde de signaler les messages en spoiler? Ou c'est ok si je retire et laisse juste comme ça ?